### ISSUE TITLE
ARM64 Linux openjdk test failures on jdk11

### ISSUE THREAD

**Issue by adam-thorpe** [AdoptOpenJDK/openjdk-tests#1499]
So I've just noticed the huge number of test failures on ARM64 openj9 builds, most of which are timeouts. I know that this is a WIP by the openj9 team but should these tests still be running (now on large heap as well) or should they be triaged? Theres roughly 50 failing tests on the latest job (but have to be manually recorded as the whole job is timing out).

eg.
https://ci.adoptopenjdk.net/user/adam-thorpe/my-views/view/Nightly%20test%20openjdk%20openJ9/job/Test_openjdk11_j9_sanity.openjdk_aarch64_linux/ (hasn't ran for a while due to other build problems)
https://ci.adoptopenjdk.net/user/adam-thorpe/my-views/view/Nightly%20test%20openjdk%20openJ9/job/Test_openjdk11_j9_sanity.openjdk_aarch64_linux_xl/

**Comment by smlambert** [comment]
Slow runs/time outs are related to this build not having JIT support yet (which is being actively worked on).  We can temporarily increase the timeout of these runs to see how long they take to complete.

**Comment by smlambert** [comment]
I have increased timeout to 20hrs, just to see how long it takes now (without a JIT).

**Comment by adam-thorpe** [comment]
Yes, my only concern was it chewing up machine time since both of these jobs haven't ran at the same time yet :)

**Comment by adam-thorpe** [comment]
@ShelleyLambert did you increase the timeout for the XL build as well? It seems to have stopped at 10 hours again.

**Comment by smlambert** [comment]
No I had not, did now (but to reiterate, this is a temporary change to see how long they run.  I had also not seen that somehow we are down a lot of aarch64 test machines, if that remains the case, I will be disabling some of this testing until we have machines back up and running).
